frontend-react/src/utils/OrganizationUtils.ts

```diff
@@ -90,7 +88,7 @@ const getRSOrgs = (
     return filterGroups().map((group) => parseOrgName(group));
 };
 
-function parseOrgs(orgs: Array<string>): Array<Partial<SessionStore>> {
+function parseOrgs(orgs: Array<string>): any[] {
```

Comment:
Is there any specificity that can be brought to this type definition? Does it even need to exist now that we have `useOktaMemberships`?